packages/video_player/video_player_android/lib/src/android_video_player.dart

```diff
@@ -307,6 +349,41 @@ class _PlayerInstance {
     return _eventStreamController.stream;
   }
 
+  Future<NativeAudioTrackData> getAudioTracks() {
+    return _api.getAudioTracks();
+  }
+
+  Future<void> selectAudioTrack(String trackId) async {
+    // Parse the trackId to get groupIndex and trackIndex
+    final List<String> parts = trackId.split('_');
+    if (parts.length != 2) {
+      throw ArgumentError(
+        'Invalid trackId format: "$trackId". Expected format: "groupIndex_trackIndex"',
+      );
+    }
+
+    final int groupIndex = int.parse(parts[0]);
+    final int trackIndex = int.parse(parts[1]);
+
+    // Create a completer to wait for the track selection to complete
+    _audioTrackSelectionCompleter = Completer<void>();
+
+    try {
+      await _api.selectAudioTrack(groupIndex, trackIndex);
+
+      // Wait for the onTracksChanged event from ExoPlayer with a timeout
```

Comment:
I would argue we don't need the `Completer` or any of this logic below. Maybe you've already discussed implementing it this way on other platforms, in which case we can dismiss my comment. But, I think it's ok to just make the call to `_api.selectAudioTrack` and use the callback on the native side to just report if there were any issues selecting the track. Thoughts?